clang/lib/CIR/CodeGen/CIRGenBuiltinAArch64.cpp

```diff
@@ -2226,13 +2226,13 @@ static mlir::Value vecReduceIntValue(CIRGenFunction &cgf, mlir::Value val,
       loc, val, builder.getConstInt(loc, cgf.SizeTy, 0));
 }
 
-mlir::Value emitNeonCall(CIRGenBuilderTy &builder,
```

Comment:
You should keep the `emitNeonCall` version, which is a thin wrapper around calling `emitNeonCallToOp<cir::LLVMIntrinsicCallOp>`, `LLVMIntrinsicCallOp` is still the majority of the calls and should be simple to call and shouldn't change existing call sites.